src/django/api/models.py

```diff
@@ -1190,3 +1190,26 @@ class ProductionType(models.Model):
         blank=False,
         help_text='A suggested value for production type'
     )
+
+
+class DownloadLog(models.Model):
+    """
+    Log CSV download requests from the web client
+    """
+    user = models.ForeignKey(
+        'User',
+        null=False,
+        on_delete=models.CASCADE,
+        help_text='The User account that made the request'
+    )
+    path = models.CharField(
+        max_length=2083,
```

Comment:
Is 2083 a maximum value for paths?

Reply:
There is a well researched StackOverflow answer that contributed to this choice of maximum value
https://stackoverflow.com/a/417184

In summary, URLs larger than this bump into some limits in browsers and search crawling infrastructure so it is a reasonable maximum value.